packages/react/src/CTABanner/CTABanner.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {forwardRef, type Ref} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+import {Heading} from '../Heading'
+import {Text} from '../Text'
+
+/**
+ * Design tokens
+ */
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/cta-banner/colors-with-modes.css'
+
+/** * Main Stylesheet (as a CSS Module) */
+import styles from './CTABanner.module.css'
+
+export type CTABannerProps = BaseProps<HTMLDivElement> & {
+  children: React.ReactNode | React.ReactNode[]
+  align?: 'start' | 'center'
+  hasBorder?: boolean
+  hasShadow?: boolean
+  hasBackground?: boolean
+}
+
+export const Root = forwardRef(
+  (
+    {
+      align = 'start',
+      hasBorder = false,
+      hasShadow = true,
+      hasBackground = true,
+      className,
+      children,
+      ...props
+    }: CTABannerProps,
+    ref: Ref<HTMLDivElement>
+  ) => {
+    return (
+      <section ref={ref} className={clsx(styles.CTABanner, hasShadow && styles['CTABanner--shadow'])} {...props}>
+        <div
+          className={clsx(
+            styles['CTABanner-container'],
+            hasBorder && styles['CTABanner-container--border'],
+            hasBackground && styles['CTABanner-container--background']
+          )}
+        >
+          <div className={clsx(styles['CTABanner-content'], align === 'center' && styles['CTABanner-content--center'])}>
+            {children}
+          </div>
+        </div>
+      </section>
+    )
+  }
+)
+
+export const CTABannerHeadingTags = ['h1', 'h2'] as const
```

Comment:
I don't think this would ever be used as a H1 or H2 given where we expect it to appear (near the bottom of the page)... 

Is there a reason for constraining it, rather than forwarding the types from `Heading`? I think setting some defaults is fine, but we should also allow app-side customisation. We've received related feedback around the River.

Reply:
This was the feedback given to me by Ash. We can open it up for sure. It was that way before and he Ash had me restrict it.

Reply:
I think using an `h3` by default would make sense, as the banner will usually be part of a section on the page, which will likely already have an `h2`.

Reply:
Ok this will change the size of the text within the banner. Should I make it visually something else?

Reply:
yeah, you can use the `size` prop to visually lock to "2" (or whatever it was before)